pkg/ring/basic_lifecycler.go

```diff
@@ -237,7 +247,7 @@ func (l *BasicLifecycler) registerInstance(ctx context.Context) error {
 		var exists bool
 		instanceDesc, exists = ringDesc.Ingesters[l.cfg.ID]
 		if exists {
-			level.Info(l.logger).Log("msg", "instance found in the ring", "instance", l.cfg.ID, "ring", l.ringName, "state", instanceDesc.GetState(), "tokens", len(instanceDesc.GetTokens()))
+			level.Info(l.logger).Log("msg", "instance found in the ring", "instance", l.cfg.ID, "ring", l.ringName, "state", instanceDesc.GetState(), "tokens", len(instanceDesc.GetTokens()), "registered_timestamp", instanceDesc.RegisteredTimestamp)
```

Comment:
Can we also include formatted date in the log message?

Reply:
We can, but could be zero value. That's why I used the epoch. WDYT?

Reply:
Anyway, the `.String()` of a zero value timestamp is `0001-01-01 00:00:00 +0000 UTC` so I guess it's quite clear to read that case.